EXTCAP.md

```diff
@@ -0,0 +1,180 @@
+# Spinel Extcap Reference
+The Spinel Extcap provides a more user-friendly way to use OpenThread Sniffer. Spinel 
+Extcap is primarily targeted for integrating OpenThread Sniffer with Wiresahrk,  and 
+is suitable for user to use OpenThread Sniffer by Wireshark GUI.
+
+## System Requirements
```

Comment:
All level 2 and below headers should be in sentence case, with only the first word capitalized unless other words are proper nouns (like Wireshark).  Review all level 2 and below headers in the document and update as needed.